src/models/invitation.model.ts

```diff
@@ -27,14 +27,18 @@ const InvitationSchema = new Schema({
     {
       email: {
         type: String,
-        requried: false,
+        required: true,
       },
       role: {
         type: String,
         enum: ROLE,
-        default: ROLE.TRAINEE, 
+        default: ROLE.TRAINEE,
+        required: true,
       },
-            
+      name: {
+        type: String,
+        required: true,
+      }     
```

Comment:
while inviting the user, only the email and role are provided, I think you can remove this.

Reply:
Now send Notification MUTATION is corrected. You can check it.

Reply:
Add the name field to the InviteeInput type in invitationSchema as well, as it is the input type passed to the resolver.